AdminUi/src/AdminUi/ClientApp/src/app/utils/number-filter.ts

```diff
@@ -0,0 +1,4 @@
+export interface NumberFilter {
+    operator?: string;
+    value?: number | null;
```

Comment:
Why is it optional AND can be null?

Reply:
Field is optional due to the usage of ngIf directives (meaning it's undefined when the filter is not rendered).
Can be null because <input> fields set the value to null when their content is deleted.

Reply:
> Can be null because fields set the value to null when their content is deleted.

Why don't they just set the value to undefined?

Reply:
I'm not sure, that was my initial thought, especially when considering I had the field set as optional only.
But the reality is, even when value was defined as `value?: number;`, it would be set as null by the input component.